src/main/java/com/ibm/cloudant/kafka/connect/CloudantSourceTaskConfig.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
```

Comment:
Remove extra * at beginning and end of copyright comment block, update line below:
```
* Copyright © 2016, 2022 IBM Corp. All rights reserved.
```